src/flightlog_fielddefs.js

```diff
@@ -143,6 +143,7 @@ export const FLIGHT_LOG_FLIGHT_MODE_NAME_POST_4_5 = makeReadOnly([
   "HEADFREE",
   "PASSTHRU",
   "FAILSAFE",
+  "POSHOLD",
```

Comment:
Think we need to revisit FM's added from 4.5. and add them instead to

https://github.com/betaflight/blackbox-log-viewer/pull/788/files#diff-1c35af2d073b48014296ed8da03ed89c112f28d05456355d2bf04f3fb3e92ce4R538-R540

Reply:
We have new AltHold and PosHold modes after 4.5. Its are included in modes list now.

Reply:
These flight modes were added in 4.6. Are they backwards compatible using 4.5 logs ?

Reply:
The new AltaHold and PosHold modes names are added for BF 4.6 FW versions only.
The previouse BF versions use own names list.